check-required.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Exit immediately, but don't expose any environment variable values
+set +x
+set -e
+
+if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]; then
+	echo "TERMINUS_SITE and TERMINUS_ENV environment variables must be set"
+	exit 1
+fi
+
+if [ -z "$WORDPRESS_ADMIN_USERNAME" ] || [ -z "$WORDPRESS_ADMIN_PASSWORD" ]; then
+	echo "WORDPRESS_ADMIN_USERNAME and WORDPRESS_ADMIN_PASSWORD environment variables must be set"
+	exit 1
+fi
+
+# Builds from forks will not have the Terminus token available.
+# Scripts should exit, but not with an error.
+terminus auth whoami > /dev/null
```

Comment:
Yay! Different fails https://circleci.com/gh/pantheon-systems/pantheon-wordpress-upstream-tests/86

Do `exit` statements cascade from one file to the next? I don't think they do. They above `exit` calls have the desire impact because the calling script has `set -ex`